x-pack/plugins/maps/public/reducers/store.js

```diff
@@ -17,11 +18,21 @@ export const DEFAULT_MAP_STORE_STATE = {
 
 export function createMapStore() {
   const enhancers = [applyMiddleware(thunk)];
-  const rootReducer = combineReducers({
+  const combinedReducers = combineReducers({
     map,
     ui,
     nonSerializableInstances,
   });
+
+  const rootReducer = (state, action) => {
```

Comment:
Why was this needed. Not sure we should ever set state to undefined. Maybe set state to initial redux state.